linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/Emitter.scala

```diff
@@ -819,29 +819,36 @@ private object Emitter {
 
     def assumingES6: Boolean = coreSpec.esFeatures.useECMAScript2015
 
+    val StringArgConstructorName = newMethodName("init___T")
+
     multiple(
-        instantiateClass("O", "init___"),
-        classData("O"),
+        instantiateClass(ObjectClass, NoArgConstructorName),
+        classData(ObjectClass),
 
-        instantiateClass("jl_CloneNotSupportedException", "init___"),
+        instantiateClass(newClassName("jl_CloneNotSupportedException"),
```

Comment:
Is there a reason these class names are not in constants? It seems that everywhere else they go in constants (even if only used once).